projects/packages/forms/src/dashboard/components/InboxStatusToggle/index.js

```diff
@@ -66,11 +65,14 @@ export default function InboxStatusToggle( { currentQuery } ) {
 
 	return (
 		<ToggleGroupControl
+			__next40pxDefaultSize
+			__nextHasNoMarginBottom
+			style={ { marginBottom: '8px' } } // Counters empty space in DataViews above this component
```

Comment:
There's an empty component above this component, causing toggle to not be center-aligned with other elements:

<img width="1301" alt="Screenshot 2025-05-29 at 12 17 53" src="https://github.com/user-attachments/assets/dae59fe3-d4b8-40a8-b4e9-78a8b2fbca4c" />

It's a shame to add but bottom margin fixes it.

Reply:
cc @oandregal @ntsekouras  if you've seen above issue before.

Reply:
This seems to be just the label container for `ToggleGroupControl`. In general you should add a `label` and you can hide it from the UI with `hideLabelFromVision` prop.